README.md

```diff
@@ -190,9 +192,8 @@ Replace the `<hostinflux>` placeholder with the hostname for a running InfluxDB
 ```
 sudo cp conf/nuodb.conf /etc/telegraf/telegraf.d
 sudo cp conf/outputs.conf /etc/telegraf/telegraf.d
-sudo edit /etc/telegraf/telegraf.conf
```

Comment:
I'm not sure what this command is for, but I didn't seem to need it. This is supposed to interactively edit the file, but I don't see what change we need to make to it.